packages/salesforcedx-vscode-apex-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -426,6 +422,166 @@ export class CheckpointService implements TreeDataProvider<BaseNode> {
+                'checkpoint_creation_status_org_info'
+              )}`
+            );
+            progress.report({
+              increment: 0,
+              message: localizedProgressMessage
+            });
+            const orgInfoRetrieved: boolean = await checkpointService.retrieveOrgInfo();
+            if (!orgInfoRetrieved) {
+              updateError = true;
+              return false;
+            }
+
+            writeToDebuggerOutputWindow(
+              `${localizedProgressMessage}, ${nls.localize(
+                'checkpoint_creation_status_source_line_info'
+              )}`
+            );
+            progress.report({
+              increment: 20,
+              message: localizedProgressMessage
+            });
+            const sourceLineInfoRetrieved: boolean = await retrieveLineBreakpointInfo();
+            // If we didn't get the source line information that'll be reported at that time, just return
+            if (!sourceLineInfoRetrieved) {
+              updateError = true;
+              return false;
+            }
+
+            // There can be a max of five active checkpoints
+            if (!checkpointService.hasFiveOrLessActiveCheckpoints(true)) {
+              updateError = true;
+              return false;
+            }
+
+            writeToDebuggerOutputWindow(
+              `${localizedProgressMessage}, ${nls.localize(
+                'checkpoint_creation_status_setting_typeref'
+              )}`
+            );
+            progress.report({
+              increment: 50,
+              message: localizedProgressMessage
+            });
+            // For the active checkpoints set the typeRefs using the source/line info
+            if (!setTypeRefsForEnabledCheckpoints()) {
+              updateError = true;
+              return false;
+            }
+
+            writeToDebuggerOutputWindow(
+              `${localizedProgressMessage}, ${nls.localize(
+                'checkpoint_creation_status_clearing_existing_checkpoints'
+              )}`
+            );
+            progress.report({
+              increment: 50,
+              message: localizedProgressMessage
+            });
+            // remove any existing checkpoints on the server
+            const allRemoved: boolean = await checkpointService.clearExistingCheckpoints();
+            if (!allRemoved) {
+              updateError = true;
+              return false;
+            }
+
+            writeToDebuggerOutputWindow(
+              `${localizedProgressMessage}, ${nls.localize(
+                'checkpoint_creation_status_uploading_checkpoints'
+              )}`
+            );
+            progress.report({
+              increment: 70,
+              message: localizedProgressMessage
+            });
+            // This should probably be batched but it makes dealing with errors kind of a pain
+            for (const cpNode of checkpointService.getChildren() as CheckpointNode[]) {
```

Comment:
I'm not a fan of abbreviating variables, even when the code is only a few lines line (as a reader it always makes me have to got back and look at what it is).  How about instead of `cpNode` it be named `checkpoiuntNode`?